ruby/generate-website-screenshot/README.md

```diff
@@ -0,0 +1,57 @@
+# 🌐 generateWebsiteScreenshot() Function using Ruby
+
+Generate a PNG screenshot of a website and return in base64 format.
+
+_Example input:_
+
+```json
+{
+    "variables": {"SCREENSHOT_API_KEY": "SCREENSHOT_API_KEY"},
+    "payload": "{\"url\":\"https://google.com\"}"
+}
+```
+
+_Example output:_
+
+```json
+{
+  "success": true,
+  "screenshot": "iVBORw0KGgoAAAANSUhEUgAAAaQAAALiCAY...QoH9hbkTPQAAAABJRU5ErkJggg=="
+}
+```
+
+## 📝 Environment Variables
+
+List of environment variables used by this cloud function:
```

Comment:
Please remove references to environment

Reply:
I meant to only remove the "environment" words because the variables are technically not environment variables.

Reply:
Technically yes you are right. I will make those changes accordingly